[SC-0XX] Add 3.2.2.4.22 DNS Change with Static Value and update 3.2.2.4.7 (#3)
slghtr-says/servercert: docs/BR.md

CAs performing validations using this method MUST implement Multi-Perspective Issuance Corroboration as specified in [Section 3.2.2.9](#3229-multi-perspective-issuance-corroboration). To count as corroborating, a Network Perspective MUST observe the same challenge information (i.e. Random Value or Request Token) as the Primary Network Perspective.

> If the CA or an Affiliate of the CA operates a DNS zone to which Applicants can delegate (via CNAME) their underscore-prefixed Domain Label, the CA MUST ensure that each Applicant delegates to a unique FQDN within that zone. A CA or Affiliate of a CA SHOULD NOT operate such a service, and SHOULD direct any Applicants using such a service to use the method described in [Section 3.2.2.4.22](#322422-dns-change-with-static-value) instead.

Author: ```suggestion
If the CA or an Affiliate of the CA operates a DNS zone to which Applicants can delegate (via CNAME) their underscore-prefixed Domain Label, the CA MUST ensure that each Applicant delegates to a unique FQDN within that zone. A CA or Affiliate of a CA SHOULD NOT operate such a service, and SHOULD direct any Applicants using such a service to use the method described in [Section 3.2.2.4.22](#322422-dns-change-with-persistent-value) instead.
```

**Note**: Once the FQDN has been validated using this method, the CA MAY also issue Certificates for other FQDNs that end with all the Domain Labels of the validated FQDN. This method is suitable for validating Wildcard Domain Names.

> ##### 3.2.2.4.22 DNS Change with Static Value

Reviewer: ```suggestion
##### 3.2.2.4.22 DNS Record with Static Value
```

> Confirming the Applicant's control over a FQDN by confirming the presence of a durable record identifying the Applicant in a DNS TXT record for an Authorization Domain Name that is prefixed with a Domain Label that begins with an underscore character.

Reviewer 2: ```suggestion
Confirming the Applicant's control over a FQDN by confirming the presence of a durable record identifying the Applicant in a DNS TXT record for an Authorization Domain Name that is prefixed with a DNS TXT Record Static DCV Domain Label.
```

I would like to suggest the above change and incorporating a new Definition: 
"DNS TXT Record Static DCV: The static DCV Domain Label described in [Appendix A.2.3]"

Then adding to Appendix A.2:

"A.2.2. DNS TXT Record Static DCV
The DNS TXT record MUST be placed on the "_validation-static" subdomain of the domain being validated."

Thoughts?

Author: @XolphinMartijn - I really like that suggestion for the consistency it provides with the other DNS TXT record based methods.

Author: For consistency with the other DNS TXT methods, perhaps we could define a ** DNS TXT Applicant Authorization Record ** which contains a list of ** Applicant Authorizations  ** (Not sold on the names, open for suggestions!) 

This would look something like: 

```suggestion
Confirming the Applicant's control over a FQDN by confirming the presence of an Applicant Authorization in a DNS TXT Applicant Authorization Record for an Authorization Domain Name.
```

where 
```
**Applicant Authorization**: The following whitespace-delimited tokens:
- ca="X", where X is an Issuer Domain Name disclosed by the CA in Section 4.2 of the CA's Certificate Policy and/or Certification Practices Statement;
- accounturi="Y", where Y is a unique URI (as described by RFC 8657, Section 3) identifying the account of the Applicant which requested validation for this FQDN.

"**DNS TXT Applicant Authorization Record**: The DNS TXT Applicant Authorization Record described in [Appendix A.2.3]"

```

and 

```
A.2.3. DNS TXT Applicant Authorization Record
The DNS TXT record MUST be placed on the "_validation-static" subdomain of the domain being validated. The entire RDATA value of this TXT record MUST consist of one or more valid Applicant Authorizations, or else it cannot be used." 
```

Basic premise is to distinguish the DNS TXT Record (and where it needs to be located) from the RDATA value (which is the Applicant specific authorization)

Thoughts?

Author: Updated with suggested changes. Determined that based on proposed language updates from @CBonnell the additional complexity isn't required.

> The CA MUST confirm the presence of a record containing the following whitespace-delimited tokens:

Reviewer: ```suggestion
The CA MUST confirm the presence of a TXT record containing the following whitespace-delimited tokens:
```

> The CA MUST confirm the presence of a record containing the following whitespace-delimited tokens:

Reviewer 3: It would be nice and consistent if the CAA issue property syntax were used here. For example:

`X; accounturi=https://x.y.z/123456`

Author: > It would be nice and consistent if the CAA issue property syntax were used here. For example:
> 
> `X; accounturi=https://x.y.z/123456`

This is a really interesting idea that I don't want to get lost in the revisions. I can see the benefit of leveraging both the CA and AccountURI syntax from CAAs since there is really no reason to introduce two different ways to express the same exact thing.

Reviewer 3: @slghtr-says let me know if you want me to submit a PR with a concrete proposal for switching to CAA issue syntax.

Author: That would be very much appreciated @CBonnell!

Reviewer 3: PR here: https://github.com/slghtr-says/servercert/pull/4

- ca="X", where X is an Issuer Domain Name disclosed by the CA in Section 4.2 of the CA's Certificate Policy and/or Certification Practices Statement;
- accounturi="Y", where Y is a unique URI (as described by RFC 8657, Section 3) identifying the account of the Applicant which requested validation for this FQDN.

For example, the DNS TXT record might look like:

> TXT ca="example.com" accounturi="https://example.com/acct/123"

Reviewer: ````suggestion
```
TXT ca="example.com" accounturi="https://example.com/acct/123"
```
````

Author: We will need to support multiple of these "token sets" inside one or more TXT record (depending on what a DNS Provider allows) to support multi-CA and multi-account use cases. What should we refer to each applicant / ca authorization individually and collectively? 

@CBonnell - Leveraging the CAA syntax may be helpful here as you suggested.

Author: An alternative approach would be to lean even further into the DNS-ACCOUNT-01 framing. 

In this paradigm, a TXT record is scoped to a single account uri but we also scope the dns name to the same account uri using the same pattern that is called out in https://datatracker.ietf.org/doc/draft-ietf-acme-dns-account-label/

`       "_" || base32(SHA-256(<ACCOUNT_URI>)[0:10]) || "._ca-validation-static" `

this would create a two level domain structure that would effectively look like (using the example from the ietf draft):

`_ujmmovf2vn55tgye._ca-validation-static.example.org`

This approach would leverage the innovation introduced in DNS-ACCOUNT-01 to allow multiple unique accounts to exist under a common uniquely scoped subdomain and avoid CA name collisions.

thoughts?

Reviewer: It's not immediately obvious to me that this needs to be as flexible as dns-account-01. That method needs to be flexible because multiple independent ACME clients might be fighting over the CNAME delegation of the original underscore-prefixed subdomain. But with static validation, it's not expected that the client will actually need to be manipulating DNS at validation time. So I'm not sure that we actually need to be able to use a CNAME to delegate DNS challenge resolution to multiple different client-controlled nameservers.

It's the same amount of work to set up a static `_sha256-of-kid._acme-challenge.example.com in CNAME` record as it is to set up a static `_ca-validation-static.example.com in TXT` record, and both of those operations are assumed to happen once and then remain in place (i.e. without being directly manipulated by the ACME client) for an extended period.

Author: @aarongable - Thanks! I buy your argument that a DNS-ACCOUNT-01 approach may not be appropriate for this method. 

I also think I initially misunderstood @XolphinMartijn statement about how to support multiple TXT records. 

My recommendation is to change the format of the account authorization string to wrap the entire line in double quotes to make it more DNS resolver agnostic. This should  allow multiple records to be added under the same name with different values regardless of how the DNS provider facilities that. For example, this should work if the DNS provider requires that TXT records are created under a single name with multiple values encased in double quotes (https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/ResourceRecordTypes.html#TXTformat-limits) or as multiple name value pairs (https://www.cloudflare.com/learning/dns/dns-records/dns-txt-record/)

Example: 
```
_validation-static.example.com IN TXT "ca=example.com;accounturi=https://example.com/acct/123"
_validation-static.example.com IN TXT "ca=example.com;accounturi=https://example.com/acct/456"
_validation-static.example.com IN TXT "ca=otherexample.com;accounturi=https://otherexample.com/acct/456"
``` 

or
```
_validation-static.example.com IN TXT "ca=example.com;accounturi=https://example.com/acct/123", "ca=example.com;accounturi=https://example.com/acct/456", 
"ca=otherexample.com;accounturi=https://otherexample.com/acct/456"
 ```

Thoughts?

Reviewer: The Route53 docs also say
> To enter more than one TXT value, enter one value per row.

So I think they support the exact same format as your CloudFlare example. And I'd expect basically all DNS providers to do the same. So I think that level of complexity isn't necessary?

Author: To be clearer, my suggestion is: 

````suggestion
```
TXT "ca=example.com;accounturi=https://example.com/acct/123"
```
````

Author: Resolving this thread in favor of @CBonnell PR https://github.com/slghtr-says/servercert/pull/4/commits/36f3c257cc6ebf954d0ea6a00f7c3ba5008d7130 which is making a similar suggestion in a clearer way.

CAs performing validations using this method MUST implement Multi-Perspective Issuance Corroboration as specified in [Section 3.2.2.9](#3229-multi-perspective-issuance-corroboration). To count as corroborating, a Network Perspective MUST observe the same challenge information as the Primary Network Perspective.

> **Note**: Once the FQDN has been validated using this method, the CA MAY also issue Certificates for other FQDNs that end with all the Domain Labels of the validated FQDN. This method is suitable for validating Wildcard Domain Names. If the CA issues a certificate for an FQDN using this method, it MUST do so within the TTL of the DNS TXT record, or 8 hours, whichever is greater.

Reviewer: I don't know if this suggestion is actually clearer; feel free to disregard if it just makes things worse :P

```suggestion
**Note**: Once the FQDN has been validated using this method, the CA MAY also issue Certificates for other FQDNs that end with all the Domain Labels of the validated FQDN. This method is suitable for validating Wildcard Domain Names. If the DNS TXT record has a TTL less than the validation data reuse period (see Section 4.2.1), then the CA MUST consider the validation data reuse period to be equal to the TTL or 8 hours, whichever is greater.
```

Reviewer 2: @aarongable  This looks very good and addresses my earlier concern. Thanks!

Author: Agreed. Thanks @aarongable for the suggestion!

> **Note**: Once the FQDN has been validated using this method, the CA MAY also issue Certificates for other FQDNs that end with all the Domain Labels of the validated FQDN. This method is suitable for validating Wildcard Domain Names. If the DNS TXT record has a TTL less than the validation data reuse period (see Section 4.2.1), then the CA MUST consider the validation data reuse period to be equal to the TTL or 8 hours, whichever is greater.

Author: Note: For consistency I will add link to section 4.2.1 since that is a reference to the same doc

> **Note**: Once the FQDN has been validated using this method, the CA MAY also issue Certificates for other FQDNs that end with all the Domain Labels of the validated FQDN. This method is suitable for validating Wildcard Domain Names. If the DNS TXT record has a TTL less than the validation data reuse period (see Section 4.2.1), then the CA MUST consider the validation data reuse period to be equal to the TTL or 8 hours, whichever is greater.

Author: ```suggestion
**Note**: Once the FQDN has been validated using this method, the CA MAY also issue Certificates for other FQDNs that end with all the Domain Labels of the validated FQDN. This method is suitable for validating Wildcard Domain Names. If the DNS TXT record has a TTL less than the validation data reuse period (see [Section 4.2.1](#421-performing-identification-and-authentication-functions)), then the CA MUST consider the validation data reuse period to be equal to the TTL or 8 hours, whichever is greater.
```

##### 3.2.2.4.22 DNS Change with Static Value

Confirming the Applicant's control over a FQDN by confirming the presence of a durable record identifying the Applicant in a DNS TXT record for an Authorization Domain Name.

The CA MUST confirm the presence of a record containing the following whitespace-delimited tokens:
- "ca=X", where X is an Issuer Domain Name disclosed by the CA in Section 4.2 of the CA's Certificate Policy and/or Certification Practices Statement;
- "accounturi=X", where X is a unique URI (as described by RFC 8657, Section 3) identifying the account of the Applicant which requested validation for this FQDN; and
- optionally "pubkey=X", where X is a base64-encoded DER public key (i.e. the body of a PEM "BEGIN PUBLIC KEY" block, with headers and whitespace removed) matching the public key provided by the Applicant in their certificate request.

For example, the record might look like (using the EC P256 test key from RFC 9500, Section 2.3):

> TXT ca=example.com accounturi=https://example.com/acct/123 pubkey=MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEQiVI+I+3gv+17KN0RFLHKh5Vj71vc75eSOkyMsxFxbFsTNEMTLjVuKFxOelIgsiZJXKZNCX0FBmrfpCkKklCcg==

Reviewer 4: nit: in RFC 1464, values are enclosed in quotes, i.e. key="value" It seems inevitable that people will quote and not quote, but it probably makes sense to explicit about a preferred format.

Author: Thanks! Addressed in follow-up commit.

**Note**: Once the FQDN has been validated using this method, the CA MAY also issue Certificates for other FQDNs that end with all the Domain Labels of the validated FQDN. This method is suitable for validating Wildcard Domain Names.

> ##### 3.2.2.4.22 DNS TXT Record with Static Value

Author: ```suggestion
##### 3.2.2.4.22 DNS TXT Record with Persistent Value
```

> Confirming the Applicant's control over a FQDN by confirming the presence of a durable record identifying the Applicant in a DNS TXT record for an Authorization Domain Name that is prefixed with one DNS TXT Record Static DCV Domain Label.

Author: ```suggestion
Confirming the Applicant's control over a FQDN by confirming the presence of a persistent record identifying the Applicant in a DNS TXT record for an Authorization Domain Name that is prefixed with one DNS TXT Record Persistent DCV Domain Label.
```

**Note**: Once the FQDN has been validated using this method, the CA MAY also issue Certificates for other FQDNs that end with all the Domain Labels of the validated FQDN. This method is suitable for validating Wildcard Domain Names.

##### 3.2.2.4.22 DNS TXT Record with Persistent Value

Confirming the Applicant's control over a FQDN by confirming the presence of a persistent record identifying the Applicant in a DNS TXT record for an Authorization Domain Name that is prefixed with one DNS TXT Record Persistent DCV Domain Label.

The CA MUST confirm the presence of a TXT record whose RDATA value fulfills the following requirements:

1. The RDATA value MUST conform to the `issue-value` syntax as defined in RFC 8659, section 4.2; and
2. The `issuer-domain-name` value MUST be an Issuer Domain Name disclosed by the CA in Section 4.2 of the CA's Certificate Policy and/or Certification Practices Statement; and
3. The `issue-value` MUST contain an `accounturi` parameter, where the parameter value is a unique URI (as described by RFC 8657, Section 3) identifying the account of the Applicant which requested validation for this FQDN.

> For example, the DNS TXT record might look like:

Author: ```suggestion
For example, the DNS TXT RDATA value might look like:
```

> TXT "example.com; accounturi=https://example.com/acct/123"

Reviewer 5: The use of example,.com as the issuer name could be misleading as we often think of that as the domain being verified.  Could we use something more descriptive such as IssuingCA.com or issuer-domain-name instead?

Reviewer 6: > The use of example,.com as the issuer name could be misleading as we often think of that as the domain being verified. Could we use something more descriptive such as IssuingCA.com or issuer-domain-name instead?

I agree that, BR should be more accessible content w/o lossing professionalism, it is helpful for CAs to avoid mistaken.

Reviewer 3: I used "authority.example" in RFC 9495 at someone's (I forget who) suggestion. Using that issuer-domain-name should make it clear it's the CA identifier and not the Applicant's identifier.

> TXT "example.com; accounturi=https://example.com/acct/123"

Author: ```suggestion
TXT "authority.example; accounturi=https://authority.example/acct/123"
```

> TXT "authority.example; accounturi=https://authority.example/acct/123"

Author: ```suggestion
_validation-persist.example.com IN TXT "authority.example; accounturi=https://authority.example/acct/123"
```

> For example, the DNS TXT record might look like:
> TXT "authority.example; accounturi=https://authority.example/acct/123"

Author: ```suggestion
For example, the DNS TXT record might look like:
_validation-persist.example.com IN TXT "authority.example; accounturi=https://authority.example/acct/123"
```

**Note**: Once the FQDN has been validated using this method, the CA MAY also issue Certificates for other FQDNs that end with all the Domain Labels of the validated FQDN. This method is suitable for validating Wildcard Domain Names.

##### 3.2.2.4.22 DNS TXT Record with Persistent Value

Confirming the Applicant's control over a FQDN by confirming the presence of a persistent record identifying the Applicant in a DNS TXT record for an Authorization Domain Name that is prefixed with one DNS TXT Record Persistent DCV Domain Label.

The CA MUST confirm the presence of a TXT record whose RDATA value fulfills the following requirements:

1. The RDATA value MUST conform to the `issue-value` syntax as defined in RFC 8659, section 4.2; and
2. The `issuer-domain-name` value MUST be an Issuer Domain Name disclosed by the CA in Section 4.2 of the CA's Certificate Policy and/or Certification Practices Statement; and

> 3. The `issue-value` MUST contain an `accounturi` parameter, where the parameter value is a unique URI (as described by RFC 8657, Section 3) identifying the account of the Applicant which requested validation for this FQDN.

Reviewer 7: ```suggestion
3. The `issue-value` MUST contain an `accounturi` parameter, where the parameter value is a unique URI (as described by RFC 8657, Section 3) identifying the account of the Applicant which requested validation for this FQDN; and
```
nit, if the suggestion below is accepted.

For example, the DNS TXT record might look like:

> _validation-persist.example.com IN TXT "authority.example; accounturi=https://authority.example/acct/123"

Reviewer 7: ```suggestion
_validation-persist.example.com IN TXT "authority.example; accounturi=https://authority.example/acct/123; expiry=1782424856"
```
example that includes the optional attribute

**Note**: Once the FQDN has been validated using this method, the CA MAY also issue Certificates for other FQDNs that end with all the Domain Labels of the validated FQDN. This method is suitable for validating Wildcard Domain Names.

##### 3.2.2.4.22 DNS TXT Record with Persistent Value

> Confirming the Applicant's control over a FQDN by confirming the presence of a persistent record identifying the Applicant in a DNS TXT record for an Authorization Domain Name that is prefixed with one DNS TXT Record Persistent DCV Domain Label.

Reviewer 5: Thanks for this great info. Regarding this statement:

- Then, when they do validation, _they'll crawl the DNS tree upwards_ to see if they find the persistent record on the requested FQDN or any of its parent ADNs. If they do, they'll mark that particular Authorization as validated and reuse it for future requests for that exact FQDN, but not for other FQDNs that share the same ADN.

The CA uses MPIC to find and validate the domain that has the persistent record as validated after walking up the chain and set the applicable "expiration date" or reuse period.  This might mean multiple MPIC lookups, one looking for a valid domain and then for the persistent record at each level as they walk up the tree to the Base Domain.  When in the process/protocol will this happen?  If it's as a response to the New Order Request, doing all the lookups might take "a while" and the whole time the ACME client has a connection open, or, more likely, I'm probably not thinking about the actual flow correctly.

> Confirming the Applicant's control over a FQDN by confirming the presence of a persistent record identifying the Applicant in a DNS TXT record for an Authorization Domain Name that is prefixed with one DNS TXT Record Persistent DCV Domain Label.

Reviewer: Probably each MPIC perspective would do the upwards tree-walking on its own. The inputs to each perspective should (imo) be the ACME Identifier being validated, not a pre-processed version thereof (since that pre-processing wouldn't have happened from multiple perspectives).

> Confirming the Applicant's control over a FQDN by confirming the presence of a persistent record identifying the Applicant in a DNS TXT record for an Authorization Domain Name that is prefixed with one DNS TXT Record Persistent DCV Domain Label.

Reviewer 3: > But an ACME server can't attach redundant authorizations to an order. If they want to give the client the option of doing a DNS-01 challenge at an ADN, or doing individual HTTP-01 challenges for each individual identifier... there's not really a way to do that within the ACME protocol.

I think [RFC 9444](https://www.rfc-editor.org/rfc/rfc9444.html) defines exactly how an ACME (client/server) implementation can do this. Perhaps this validation method will be the motivation for this RFC to be actually implemented 😄

> Confirming the Applicant's control over a FQDN by confirming the presence of a persistent record identifying the Applicant in a DNS TXT record for an Authorization Domain Name that is prefixed with one DNS TXT Record Persistent DCV Domain Label.

Reviewer 3: Yes, that's right. It was agreed it would be nice to have a ACME spec, but the development of that spec need not hold up progressing with this ballot.

The CA MUST confirm the presence of a TXT record whose RDATA value fulfills the following requirements:

1. The RDATA value MUST conform to the `issue-value` syntax as defined in RFC 8659, section 4.2; and
2. The `issuer-domain-name` value MUST be an Issuer Domain Name disclosed by the CA in Section 4.2 of the CA's Certificate Policy and/or Certification Practices Statement; and
3. The `issue-value` MUST contain an `accounturi` parameter, where the parameter value is a unique URI (as described by RFC 8657, Section 3) identifying the account of the Applicant which requested validation for this FQDN; and
4. The `issue-value` MAY contain an `persistUntil` parameter. If present, the parameter value MUST be a base-10 encoded integer representing a UNIX timestamp (the number of seconds since 1970-01-01T00:00:00Z).

If the `persistUntil` parameter is present, the CA MUST evaluate its value. If the time of the check is after the time specified in the `persistUntil` parameter value, the CA MUST NOT use the record as evidence of the Applicant's control over the FQDN.
For example, the DNS TXT record might look like:

> _validation-persist.example.com IN TXT "authority.example; accounturi=https://authority.example/acct/123; persistUntil=1782424856"

Author: Based on feedback from the VSC call on from 7/10 I added more examples in the ballot overview: https://github.com/slghtr-says/servercert/pull/3#issue-2899226023

Looking for suggestions on whether or not these examples (or more variations of them) should also be added directly to the ballot text and if so, suggestions on how and where.

Author: @xiaohuilam  these examples make the mechanics of the ballot any clearer?

CAs performing validations using this method MUST implement Multi-Perspective Issuance Corroboration as specified in [Section 3.2.2.9](#3229-multi-perspective-issuance-corroboration). To count as corroborating, a Network Perspective MUST observe the same challenge information as the Primary Network Perspective.

> If the DNS TXT record has a TTL less than the validation data reuse period (see [Section 4.2.1](#421-performing-identification-and-authentication-functions)), then the CA MUST consider the validation data reuse period to be equal to the TTL or 8 hours, whichever is greater.

Reviewer 5: Based on the VWG meeting on July 10th, using TTL is sufficient, especially given that domain reuse will go down to 10 days in a few years.  We can close this comment thread.

**Note**: Once the FQDN has been validated using this method, the CA MAY also issue Certificates for other FQDNs that end with all the Domain Labels of the validated FQDN. This method is suitable for validating Wildcard Domain Names.

##### 3.2.2.4.22 DNS TXT Record with Persistent Value

Confirming the Applicant's control over a FQDN by confirming the presence of a persistent record identifying the Applicant in a DNS TXT record for an Authorization Domain Name that is prefixed with one DNS TXT Record Persistent DCV Domain Label.

The CA MUST confirm the presence of a TXT record whose RDATA value fulfills the following requirements:

1. The RDATA value MUST conform to the `issue-value` syntax as defined in RFC 8659, section 4.2; and
2. The `issuer-domain-name` value MUST be an Issuer Domain Name disclosed by the CA in Section 4.2 of the CA's Certificate Policy and/or Certification Practices Statement; and
3. The `issue-value` MUST contain an `accounturi` parameter, where the parameter value is a unique URI (as described by RFC 8657, Section 3) identifying the account of the Applicant which requested validation for this FQDN; and

> 4. The `issue-value` MAY contain an `persistUntil` parameter. If present, the parameter value MUST be a base-10 encoded integer representing a UNIX timestamp (the number of seconds since 1970-01-01T00:00:00Z).

Author: ```suggestion
4. The `issue-value` MAY contain a `persistUntil` parameter. If present, the parameter value MUST be a base-10 encoded integer representing a UNIX timestamp (the number of seconds since 1970-01-01T00:00:00Z ignoring leap seconds).
```

Rationale: Align with the NumericDate definition in RFC7519 https://datatracker.ietf.org/doc/html/rfc7519#section-2 by clarifying that leap seconds should be ignored.

The DNS TXT record MUST be placed on the "`_validation-contactphone`" subdomain of the domain being validated. The entire RDATA value of this TXT record MUST be a valid Global Number as defined in RFC 3966, Section 5.1.4, or it cannot be used.

### A.2.3. DNS TXT Record Persistent DCV Domain Label

> The DNS TXT record MUST be placed on the "`_validation-persist`" subdomain of the domain being validated.

Reviewer 8: FWIW, we do use "DNS zone" in the new language introduced by this ballot, so in part I was thinking of consistency with that, but I like the language you've proposed here too so 🤷‍♀️

> The DNS TXT record MUST be placed on the "`_validation-persist`" subdomain of the domain being validated.

Author: ha! Great call out @clintwilson. 

Looking at the most common definition of DNS zone: 
> A DNS zone is a specific portion of the DNS namespace in the DNS, which a specific organization or administrator manages.

Per that definition, I think it still makes sense to use DNS zone in the Method 7 note  but use the "prepended label" language here in the Appendix.

> The DNS TXT record MUST be placed on the "`_validation-persist`" subdomain of the domain being validated.

Reviewer 8: Yup! To be clear, I'm satisfied with your proposed wording update as-is.
